jetty-server/src/main/java/org/eclipse/jetty/server/ServerConnector.java

```diff
@@ -348,6 +349,18 @@ protected ServerSocketChannel openAcceptChannel() throws IOException
         return serverChannel;
     }
 
+    private <T> void setSocketOption(ServerSocketChannel channel, SocketOption<T> name, T value)
+    {
+        try
+        {
+            channel.setOption(name, value);
+        }
+        catch (Throwable t)
+        {
+            LOG.warn("Unable to set socket option {} to {}", name, value, t);
```

Comment:
Don't want to warn here, see `ClientConnector.setSocketOption(...)`, just use the same debug that's there.

Reply:
Updated.